packages/ag-charts-website/src/utils/urlWithPrefix.test.ts

```diff
@@ -2,33 +2,39 @@ import { urlWithPrefix } from './urlWithPrefix';
 
 describe('urlWithPrefix', () => {
     test.each`
-        url                      | framework       | expected
-        ${'/'}                   | ${'javascript'} | ${'/ag-charts/javascript/'}
-        ${'./docs'}              | ${'javascript'} | ${'/ag-charts/javascript/docs/'}
-        ${'./with-slash/'}       | ${'javascript'} | ${'/ag-charts/javascript/with-slash/'}
-        ${'./docs'}              | ${'react'}      | ${'/ag-charts/react/docs/'}
-        ${'./docs/path'}         | ${'react'}      | ${'/ag-charts/react/docs/path/'}
-        ${'/gallery'}            | ${'react'}      | ${'/ag-charts/gallery/'}
-        ${'/with-slash/'}        | ${'javascript'} | ${'/ag-charts/with-slash/'}
-        ${'https://youtube.com'} | ${'react'}      | ${'https://youtube.com'}
-        ${'./someImage.png'}     | ${'javascript'} | ${'/ag-charts/javascript/someImage.png'}
-        ${'/someImage.png'}      | ${'javascript'} | ${'/ag-charts/someImage.png'}
+        url                           | framework       | expected
+        ${'./'}                       | ${'javascript'} | ${'/ag-charts/javascript/'}
+        ${'./docs'}                   | ${'javascript'} | ${'/ag-charts/javascript/docs/'}
+        ${'./with-slash/'}            | ${'javascript'} | ${'/ag-charts/javascript/with-slash/'}
+        ${'./docs'}                   | ${'react'}      | ${'/ag-charts/react/docs/'}
+        ${'./docs/path'}              | ${'react'}      | ${'/ag-charts/react/docs/path/'}
+        ${'/gallery'}                 | ${'react'}      | ${'/ag-charts/gallery/'}
+        ${'/'}                        | ${'javascript'} | ${'/ag-charts/'}
+        ${'/with-slash/'}             | ${'javascript'} | ${'/ag-charts/with-slash/'}
+        ${'https://youtube.com'}      | ${'react'}      | ${'https://youtube.com'}
+        ${'https://www.ag-grid.com/'} | ${'react'}      | ${'https://www.ag-grid.com/'}
+        ${'./someImage.png'}          | ${'javascript'} | ${'/ag-charts/javascript/someImage.png'}
+        ${'/someImage.png'}           | ${'javascript'} | ${'/ag-charts/someImage.png'}
+        ${'/?urlParams=something'}    | ${'javascript'} | ${'/ag-charts/?urlParams=something'}
     `('returns $expected for url $url, framework $framework siteBaseUrl /ag-charts', ({ url, framework, expected }) => {
         const siteBaseUrl = '/ag-charts';
         expect(urlWithPrefix({ url, framework, siteBaseUrl })).toBe(expected);
     });
 
     test.each`
-        url                  | framework       | expected
-        ${'/'}               | ${'javascript'} | ${'/ag-charts/javascript/'}
```

Comment:
This was incorrect before, but didn't notice

* `/` -> root
* `./` -> framework root